releng/gradle-composite/build.gradle

```diff
@@ -1,25 +1,40 @@
+apply from: 'helper.gradle'
```

Comment:
A more meaningful file name would be nice, e.g. `projects.gradle` instead of `helper.gradle`.